tests/integration/golang/helpers/client.go

```diff
@@ -109,12 +117,18 @@ func (c *HttpClient) DoRequest(uri string, values ...any) error {
 		requestBody = bytes.NewBuffer(data)
 	}
 
-	// 2. build actual URL.
-	u, err := url.Parse(fmt.Sprintf("%s%s%s", c.baseURL, c.basePath, fmt.Sprintf(uri, values...)))
+	// 2. build path with namespace.
+	path := c.basePath
```

Comment:
it is a bit strange moment. since now we have more logic related to `url` contractions, won't it be reasonable to do like this:
```
path = []string{c.baseURL}
if c.namespace != "" {
               path = append(path, "ns",  c.namespace, c.basePath)
 	}
uri, err := url.Joins(append(path, fmt.Sprintf(uri, values...)))
```
so I mean to construct final `url` like we do everywhere?